conn_test.go

```diff
@@ -73,6 +74,46 @@ func stressDuplex(t *testing.T) {
 	}
 }
 
+func TestStressClose(t *testing.T) {
```

Comment:
This test can reproduce original problem not only on Travis but local.
It doesn't detect in 100%, so `go test . -count=N` would help.